### PR TITLE
add citation info to export screen

### PR DESCRIPTION
**Pull request by VeenDuco**
Here a pull request to start a discussion on what adding citation information should look like. This relates to #1402 and #1403. 

What I did now is to add a simple box on the export screen with the citation information. This looks like the figure below. Any thoughts? 

citation_example

### CODE REVIEW

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 1]
```diff
@@ -313,7 +314,31 @@ const ExportPage = (props) => {
...
+  >
+    <TypographySubtitle1Medium>To Cite ASReview</TypographySubtitle1Medium>
+    <Typography variant="body2" sx={{ color: "text.secondary" }}>
+      ASReview LAB developers. (2023). ASReview LAB - 
+      A tool for AI-assisted systematic reviews (v1.3rc1). Zenodo.  
+      <Link href="https://doi.org/10.5281/zenodo.8297019">https://doi.org/10.5281/zenodo.8297019</Link>
+    </Typography>
+  </Box>
+  </Box>
+
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:341]: Can you prettify this file (only this file)? https://github.com/asreview/asreview/blob/master/DEVELOPMENT.md#formatting-and-linting-1

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 2]
```diff
@@ -313,7 +314,31 @@ const ExportPage = (props) => {
...
+<Box 
+  display="flex"
+  justifyContent="center"
+  alignItems="center"
+  height="100%"  // This ensures the container takes the full height of its parent
+>            
+  <Box 
+    border={2} 
+    borderColor="warning.main"
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:327]: Please use warnings for warnings :)

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks for the PR! Very welcome.  First brief review from my side.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 1 above)
Comment (VeenDuco) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js]: Done

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 3]
```diff
@@ -313,7 +314,31 @@ const ExportPage = (props) => {
...
+    borderColor="warning.main" 
+    borderRadius={3} 
+    padding={2} 
+    bgcolor="warning.lighter" 
+    maxWidth="600px"  // or whatever maximum width you prefer
+  >
+    <TypographySubtitle1Medium>To Cite ASReview</TypographySubtitle1Medium>
+    <Typography variant="body2" sx={{ color: "text.secondary" }}>
+      ASReview LAB developers. (2023). ASReview LAB - 
+      A tool for AI-assisted systematic reviews (v1.3rc1). Zenodo.
```
[the remark this answers is not in the document]
Comment (VeenDuco) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js]: I think I got this to work now. Not sure if it's an optimal solution.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 2 above)
Comment (VeenDuco) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js]: I used the info box color now.

**Comment by VeenDuco** [comment]
The requested changes should now be implemented and Rens' additions are also included.

**Comment by george-gca** [comment]
It would be really useful to have a citation version in bibtex.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-import * as React from "react";
+import React, { useState }  from 'react';
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:1]: Please apply Prettier to this file and CiteDialog.js.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 5]
```diff
@@ -130,6 +140,12 @@ const ExportPage = (props) => {
     queryClient.resetQueries("fetchDatasetWriter");
   };
 
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  const handleOpen = () => {
+    setDialogOpen(true);
+  };
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:147]: This function is available for open but not for close. It's more consistent to make this into a toggle function.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 6]
```diff
@@ -314,6 +330,59 @@ const ExportPage = (props) => {
...
+                        startIcon={<StarBorder />}
+                      >
+                        Star
+                      </Button>
+                      <Button 
+                        variant="outlined" 
+                        color="primary" 
+                        component={Link}
+                        target="_blank" 
+                        href="https://steun.uu.nl/project/asreview?locale=en"
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:368]:
```suggestion
                        href="https://asreview.nl/donate"
```

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 7]
```diff
@@ -314,6 +330,59 @@ const ExportPage = (props) => {
...
+                    >
+                      <Button 
+                        startIcon={<LibraryBooks />} 
+                        variant="outlined" 
+                        color="primary" 
+                        onClick={handleOpen}
+                      >
+                        Cite
+                      </Button>
+                      <CiteDialog isOpen={dialogOpen} onClose={() => setDialogOpen(false)} />
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:351]: See toggle function comment.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 8]
```diff
@@ -314,6 +330,59 @@ const ExportPage = (props) => {
                   </span>
                 </Tooltip>
               </Box>
+              .
+                <StyledActionsBox>
+                  <Box borderRadius={3} padding={2} maxWidth="600px">
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:335]: Do we need the width hardcoded?

File: asreview/webapp/src/Components/CiteDialog.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+    'ASReview LAB developers. ASReview LAB: A Tool for AI-Assisted Systematic Reviews [Internet]. Zenodo; 2023 [cited year month day]. Available from: https://doi.org/10.5281/zenodo.3345592'
+    ],
+    'BibTex': [
+      '@software{asreviewlab2023,',
+      'title={{ASReview LAB: A Tool for AI-Assisted Systematic Reviews}},',
+      'author={{ASReview LAB developers}},',
+      'year={2023},',
+      'url={https://doi.org/10.5281/zenodo.3345592},',
+      'note={Software version: [version number]}'
+    ],
```
Comment (J535D165) [inline asreview/webapp/src/Components/CiteDialog.js:72]: Missing closing bracket?

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 10]
```diff
@@ -332,8 +401,15 @@ const ExportPage = (props) => {
...
 };
 
-export default ExportPage;
+const mapStateToProps = (state) => {
+  return {
+    asreview_version: state.asreview_version,
+  };
+};
+
+export default connect(mapStateToProps)(ExportPage);
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:415]: This code is here to pass the version number to the component. We are not using it now. I suggest inserting it into the citation styles. See the earlier version by @VeenDuco .

File: asreview/webapp/src/Components/CiteDialog.js [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+    'RIS': [
+      'TY  - COMP',
+      'AU  - ASReview LAB developers',
+      'PY  - 2023',
+      'TI  - ASReview LAB: A Tool for AI-Assisted Systematic Reviews',
+      'PB  - Zenodo',
+      'UR  - https://doi.org/10.5281/zenodo.3345592',
+      'ER  - '
```
Comment (J535D165) [inline asreview/webapp/src/Components/CiteDialog.js:80]:
```suggestion
      'UR  - https://doi.org/10.5281/zenodo.3345592',
      'DO  - 10.5281/ZENODO.10084260',
      'ER  - '
```

**Review by J535D165 — CHANGES_REQUESTED** [review]

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 10 above)
Comment (Rensvandeschoot) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:415]: I have moved the code to the new file CiteDialog

File: asreview/webapp/src/Components/CiteDialog.js (hunk 9 above)
Comment (Rensvandeschoot) [reply asreview/webapp/src/Components/CiteDialog.js:72]: yes indeed!

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 5 above)
Comment (Rensvandeschoot) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:147]: done

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 8 above)
Comment (Rensvandeschoot) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:335]: maxWidth: { xs: '80%', sm: '40%', md: '30%' }, width: '100%'

**Comment by Rensvandeschoot** [comment]
I changed the text into buttons and added citations including BibTex (ping @george-gca ) . I only want to update the documentation with these screenshots. 

export_screen

cite

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks all!

**Pull request merged**